### ISSUE TITLE
[Question] Visited videos added to history

### ISSUE THREAD

**Issue by JediMaster25** [iv-org/invidious#3859]
When a video ends, the next one loads automatically and it gets added to the history even when I don't start playing it. Then I have to manually go to the history and delete it since I haven't actually watched it. I think the video shouldn't be added to the history unless it starts playing.

**Comment by unixfox** [comment]
> I have autoplay and autoplay next video checked, the next video doesn't play but that's another issue, it's just opened and added to watch history. I've unchecked play next to avoid this.

I don't understand what do you mean by the next video doesn't play?

Do you understand that it makes sense to add to the history a video when the video page is loaded? I mean it makes sense...

I've difficulties to understand what you really want to change here.

**Comment by JediMaster25** [comment]
> I don't understand what do you mean by the next video doesn't play?

I had `Play next by default` and `Autoplay next video` checked. The next video was loading but it wasn't playing automatically, I had to play it manually. So the second preference is not doing anything.

> Do you understand that it makes sense to add to the history a video when the video page is loaded? I mean it makes sense...

I understand that it's added to the history when the video page is loaded even if it's unexpected. I expected it to be added when I started playing or finished watching.

> I've difficulties to understand what you really want to change here.

I don't want to change anything anymore, now that I've unchecked `Play next by default` it's not adding an extra video to the history so I have no problem.
